TrkHitReco/fcl/prolog_trigger.fcl

```diff
@@ -74,10 +83,10 @@ TrkHitRecoTrigger : {
     # SEQUENCES
     # production sequence to prepare hits for tracking
     sequences: {
-	TTprepareHits     : [ TTmakeSH, TTmakePH,TTflagBkgHits ]
-	TTprepareHitsUCC  : [ TTmakeSHUCC, TTmakePHUCC, TTflagBkgHitsUCC ]
-	TTmakefastHits    : [ TTmakeSH, TTmakePH ]
-	TTSprepareHits    : [ TTmakeSH, TTSmakePH, TTmakeSTH ,TTSflagBkgHits ]
+	TTprepareHits     : [ TTPBTFSD, TTmakeSH, TTmakePH,TTflagBkgHits ]
```

Comment:
If the model is we will use a nominal value for PBT in trigger shouldh't we be using the reco PBT from EWMProducer?  You convinced me the PBTFSD was just a diagnostic.